src/status_im/data_store/activities_test.cljs

```diff
@@ -0,0 +1,95 @@
+(ns status-im.data-store.activities-test
+  (:require [cljs.test :refer [deftest is testing]]
+            [quo.design-system.colors :as colors]
+            [status-im.constants :as constants]
+            [status-im.data-store.activities :as store]))
+
+(def raw-notification
+  {:chatId                    "0x04c66155"
+   :contactVerificationStatus constants/contact-verification-state-pending
+   :lastMessage               {}
+   :name                      "0x04c661"
+   :replyMessage              {}})
+
+(deftest <-rpc-test
+  (testing "assocs random chat color"
+    (is (contains? (set colors/chat-colors) (:color (store/<-rpc raw-notification)))))
+
+  (testing "renames keys"
+    (is (= {:name                        "0x04c661"
+            :chat-id                     "0x04c66155"
+            :contact-verification-status constants/contact-verification-state-pending}
```

Comment:
Maybe it would be better to define `name` and `chat-id` in a variable since they are used multiple times.